topics/imaging/tutorials/process-image-bioimageio/tutorial.md

```diff
@@ -66,19 +66,17 @@ Here we illustrate the type of information that is both useful for understanding
 
 As an example, we consider the following model: **🧬 NucleiSegmentationBoundaryModel**
 
-This model segments nuclei in fluorescence microscopy images. It predicts <em>boundary maps</em> and <em>foreground probabilities</em> for nucleus segmentation, primarily in images stained with DAPI. The outputs are designed to be post-processed with methods such as Multicut or Watershed to achieve instance-level segmentation (object-based segmentation).
+This model segments nuclei in fluorescence microscopy images. It predicts *boundary maps* and *foreground probabilities* for nucleus segmentation, primarily in images stained with DAPI. The outputs are designed to be post-processed with methods such as Multicut or Watershed to achieve instance-level segmentation (object-based segmentation).
 
 - **Imaging modality**: Fluorescence microscopy
 - **Task**: Nucleus segmentation (boundary-aware)
-- **Input axes**: `bcyx`
-- **Input size**: `256,256,1,1`
```

Comment:
The explanation of those values comes only later, and without further information here, I think these values are rather confusing to the reader. Besides, at this point, I think they do not contribute to the necessary context for the understanding of the flow of the tutorial (this comes only later), hence I think we can remove this.